ui/src/app/services/review-state.service.ts

```diff
@@ -25,10 +25,11 @@ export const initialFilterState = {
       'Select All'
     ]
   },
+  // TODO Uncomment code filters once decision has been made for server-side implementation
```

Comment:
Does this go away?

Reply:
Uncommented all the code filters and switched to text filter instead of checkboxes.